godel_msgs/srv/SurfaceBlendingParameters.srv

```diff
@@ -1,8 +1,19 @@
 int32 GET_CURRENT_PARAMETERS=0
 int32 GET_DEFAULT_PARAMETERS=1
+int32 SET_PARAMETERS=2
+int32 SAVE_PARAMETERS=3
+
 int32 action
+
+godel_msgs/RobotScanParameters robot_scan
+godel_msgs/SurfaceDetectionParameters surface_detection
+godel_msgs/BlendingPlanParameters blending_plan
+godel_msgs/ScanPlanParameters scan_plan
+
 ---
+
 bool succeeded
 godel_msgs/RobotScanParameters robot_scan
 godel_msgs/SurfaceDetectionParameters surface_detection
 godel_msgs/BlendingPlanParameters blending_plan
+godel_msgs/ScanPlanParameters scan_plan
```

Comment:
Also: this looks like a getter and a setter in a single service? I was going to make the same comment about semantic naming as with `BlendProcessExecution`, but that it's a bit harder to come up with an alternative name for this one then. Perhaps `Mutate..`?